README.rst

```diff
@@ -6,9 +6,13 @@
 
 |docs| |devdocs| |usergroup| |developergroup| |anaconda| |mybinder|
 
-MDAnalysis_ is a Python library for the analysis of computer simulations of many-body systems at the molecular scale, spanning use cases from interactions of drugs with proteins to novel materials. It is widely used in the scientific community and is written by scientists for scientists. 
+.. image:: https://pbs.twimg.com/profile_images/435418713366339584/ti3KF3aU_400x400.png
```

Comment:
The image shows up too big and generally distracts from the actual content of the README. Also, avoid linking to external content. There's a logo in the repo under docs/sphinx somewhere.

However, just drop the image inclusion to keep the README easily readable.

Reply:
As far as I can tell, you didn't address my comment. The diff still contains the changes that I wanted you to reverse. Just remove the image.

Reply:
@orbeckst Did as directed!